sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -691,6 +739,58 @@ def _to_outgoing_message(self):
             to=self.to,
         )
 
+    def __repr__(self):
+        # type: () -> str
+        # pylint: disable=bare-except
+        message_repr = "body={}".format(
+            str(self.body)
+        )
+        try:
+            message_repr += ", application_properties={}".format(self.application_properties)
+        except:
+            pass
+        try:
+            message_repr += ", session_id={}".format(self.session_id)
+        except:
+            pass
+        try:
+            message_repr += ", message_id={}".format(self.message_id)
+        except:
+            pass
+        try:
+            message_repr += ", content_type={}".format(self.content_type)
+        except:
+            pass
+        try:
+            message_repr += ", correlation_id={}".format(self.correlation_id)
+        except:
+            pass
+        try:
+            message_repr += ", to={}".format(self.to)
+        except:
+            pass
+        try:
+            message_repr += ", reply_to={}".format(self.reply_to)
+        except:
+            pass
+        try:
+            message_repr += ", reply_to_session_id={}".format(self.reply_to_session_id)
+        except:
+            pass
+        try:
+            message_repr += ", subject={}".format(self.subject)
+        except:
+            pass
+        try:
+            message_repr += ", scheduled_enqueue_time_utc={}".format(self.scheduled_enqueue_time_utc)
+        except:
+            pass
+        try:
+            message_repr += ", auto_renew_error={}".format(self.auto_renew_error)
+        except:
+            pass
```

Comment:
could we use the `__repr__` of parent class ServiceBusMessage to avoid duplicate?

Reply:
I really don't think we should align with it - given repr is 1024 chars long only - i don't think gives out any information important enough

Reply:
the super class returns "ServiceBusMessage ......." as opposed to "ServiceBusReceivedMessage ....."